app/controllers/supplejack_api/concerns/stories_controller_metrics.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module SupplejackApi
+  module Concerns
+    module StoriesControllerMetrics
+      extend ActiveSupport::Concern
+      include IgnoreMetrics
+
+      included do
+        after_action :create_story_record_views, only: :show
+
+        def create_story_record_views
+          return unless @api_response[:payload] && log_request_for_metrics?
+
+          @api_response[:payload][:contents].each do |record|
```

Comment:
Check out what happens with stories that have items in them that are text embeds etc.